server.go

```diff
@@ -592,6 +593,14 @@ func (s *Server) CreateClient(w http.ResponseWriter, r *http.Request, ps httprou
 	c := s.Config.GetUserConfig(user)
 	log.Debugf("user config: %#v", c)
 
+	if *maxNumberClientConfig > 0 {
+		if len(c.Clients) >= *maxNumberClientConfig {
+			log.Error(fmt.Errorf("user %s have to many configs", c.Name))
```

Comment:
Spelling nit: "user %q has too many configs"

Reply:
Should I change that in the PR directly, or are you doing it when merging?

Reply:
Think I fixed it :)